release-calendar/src/client/components/SearchBar.tsx

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import '../stylesheets/searchBar.scss';
+import * as React from 'react';
+import {ApiService} from '../api-service';
+import Autocomplete, {RenderInputParams} from '@material-ui/lab/Autocomplete';
+import TextField from '@material-ui/core/TextField';
+
+export interface SearchBarProps {
+  handleSelectedRelease: (release: string) => void;
+}
+
+export interface SearchBarState {
+  releaseNames: string[];
+  input: string;
+  validSearch: boolean;
+}
+
+export class SearchBar extends React.Component<SearchBarProps, SearchBarState> {
+  private apiService: ApiService;
+  constructor(props: Readonly<SearchBarProps>) {
+    super(props);
+    this.state = {
+      releaseNames: [],
+      input: null,
+      validSearch: null,
+    };
+    this.apiService = new ApiService();
+    this.onChange = this.onChange.bind(this);
+    this.onSubmit = this.onSubmit.bind(this);
+    this.onClose = this.onClose.bind(this);
+    this.onInputChange = this.onInputChange.bind(this);
+  }
+
+  async componentDidMount(): Promise<void> {
+    const releaseNames = await this.apiService.getReleaseNames();
+    this.setState({releaseNames});
+  }
+
+  onSubmit(event: React.FormEvent<HTMLFormElement>): void {
+    event.preventDefault();
+  }
+
+  onClose(event: React.ChangeEvent<{}>, reason: string): void {
```

Comment:
When the search bar is cleared, the calendar still filters on the previous release.

Reply:
This problem still exists.